jsonschema/validators.py

```diff
@@ -692,14 +681,12 @@ def resolving(self, ref):
 
     def _finditem(self, schema, key):
         results = []
+        if key in schema:
+            results.append(schema)
 
-        if isinstance(schema, dict):
```

Comment:
Looks like you reintroduced the issue with handling schema is None here.

This commit history where you add code and then refactor it again in later commits makes it harder for reviewers and harder to trace the history once merged. Make your commits clean in that they do 1 thing. Put any preparatory refactoring that shouldn't change behavior and any fixes first. You're rebasing the branch anyways. As @Julian mentioned, all this won't be squashed when merged.

Reply:
Sorry for that, that happened as I tried to fix some things in order to push the coverage of the patch. I will go over the history and also fix that part.

Reply:
Fixed the mentioned issue, also I cleaned the commit history :slightly_smiling_face: